coralproject/talk
# [CORL-1355] Place comment form at bottom when sorting by oldest on featured tab

## What does this PR do?

Allows the comment form to appear at the bottom of the stream on the featured tab when `ALTERNATE_OLDEST_FIRST_VIEW` is enabled.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Enable `ALTERNATE_OLDEST_FIRST_VIEW`
- Go to featured tab
- Sort by oldest
- Go to bottom of stream
- See comment form
- Post a comment
